[MRG] Fixing bug when participants file is a subset of what mne-bids writes (#648)
mne-tools/mne-bids: doc/whats_new.rst

- Fix a bug in :func:`mne_bids.write_raw_bids`, where original format of data was not kept when writing to FIFF, by `Alexandre Gramfort`_, `Stefan Appelhoff`_, and `Richard Höchenberger`_ (:gh:`610`)
- Fix a bug where conversion to BrainVision format was done even when non-Volt channel types were present in the data (BrainVision conversion is done by ``pybv``, which currently only supports Volt channel types), by `Stefan Appelhoff`_ (:gh:`619`)
- Ensure sidecar files (`.tsv` and `.json`) are always read and written in UTF-8, by `Richard Höchenberger`_ (:gh:`625`)

> - Fix a bug where participants files were not being appended to correctly when they contained a subset of ``hand``, ``age`` and ``sex``, by `Adam Li`_ (:gh:`648`)

Reviewer: participants files or file? I thought there was only one?

> - Fix a bug where participants files were not being appended to correctly when they contained a subset of ``hand``, ``age`` and ``sex``, by `Adam Li`_ (:gh:`648`)

Author: ```suggestion
- Fix a bug where participants.tsv file was not being appended to correctly when they contained a subset of ``hand``, ``age`` and ``sex``, by `Adam Li`_ (:gh:`648`)
```

> - Fix a bug where participants.tsv file was not being appended to correctly when they contained a subset of ``hand``, ``age`` and ``sex``, by `Adam Li`_ (:gh:`648`)

Reviewer 2: ```suggestion
- Fix a bug where ``participants.tsv`` was not being appended to correctly when it contained a subset of ``hand``, ``age`` and ``sex``, by `Adam Li`_ (:gh:`648`)
```
